themes/10up-block-theme/theme.json

```diff
@@ -0,0 +1,390 @@
+{
+	"settings": {
+		"appearanceTools": true,
+		"color": {
+			"defaultDuotone": false,
+			"defaultGradients": false,
+			"defaultPalette": false,
+			"palette": [
```

Comment:
Pretty much all the tokens in here are inspired by UI Kit. I don't think we should be as opinionated in the scaffold. 

Though I would like us to ship stuff like the section styles for color theming.